configs/palm_30b_zero.py

```diff
@@ -0,0 +1,28 @@
+from colossalai.zero.shard_utils import TensorShardStrategy
+
+SEQ_LENGTH = 2048
+BATCH_SIZE = 4
+NUM_EPOCHS = 1
+# WARMUP_EPOCHS = 1
+
+parallel = dict(
+    # tensor=dict(mode="3d", size=8),
```

Comment:
is this line commented out on purpose?

Reply:
Yes, we don't need tp here. I will delete this field soon.

Reply:
ok